readme.md

```diff
@@ -19,6 +19,9 @@ $resumator = new Resumator("YOUR_API_KEY");
 // return all Jobs
 $jobs = $resumator->getJobs();
 
+// return all Jobs without caching
+$jobs = $resumator->getJobs();
```

Comment:
Ooops! I meant to delete this line. Please feel free to remove this duplication. The readme was updated explaining that caching options should be configured before API calls.